screens/auth/home/tabs/settings/components/LogoutTile.tsx

```diff
@@ -0,0 +1,60 @@
+import React, {useCallback} from 'react';
+import {
+  View,
+  StyleSheet,
+  useColorScheme,
+  TouchableHighlight,
+  Text,
+} from 'react-native';
+import auth from '@react-native-firebase/auth';
+import Icon from 'react-native-vector-icons/FontAwesome';
+
+import {getAppColorStyles} from '../../../../../../utils/styles/colors';
+import {getAppColors} from '../../../../../../utils/colors';
```

Comment:
I think those paths are becoming a bit long, so maybe it's the time to start thinking about solving it in a gentle way - [here's an idea](https://stackoverflow.com/questions/68731273/is-it-possible-to-avoid-long-relative-import-paths-on-typescript) 😄